Sources/KIF/Additions/UIWindow+KIFSwizzle.m

```diff
@@ -0,0 +1,36 @@
+//
+//  UIWindow+KIFAdditions.m
+//  KIF
+//
+//  Created by Steve Sun on 2023-04-02.
+//
+
+#import "UIWindow+KIFSwizzle.h"
+#import "UIApplication-KIFAdditions.h"
+#import <objc/runtime.h>
+
+@implementation UIWindow (KIFSwizzle)
+
++ (void)load
+{
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        Class class = [self class];
+
+        SEL originalSelector = @selector(becomeKeyWindow);
+        SEL swizzledSelector = @selector(swizzle_becomeKeyWindow);
+
+        Method originalMethod = class_getInstanceMethod(class, originalSelector);
+        Method swizzledMethod = class_getInstanceMethod(class, swizzledSelector);
+
+        method_exchangeImplementations(originalMethod, swizzledMethod);
+    });
+}
+
+- (void)swizzle_becomeKeyWindow
```

Comment:
Is there a scenario where we'd want this set in init as well, if a new window was created, but not made the keyWindow?

Reply:
@justinseanmartin perhaps we only need to set this in init? Since all available UIWindows animation speed will get set due to [this](https://github.com/kif-framework/KIF/pull/1279/files#diff-02e0ae04f5d0c0f4f624ae5086b62b1812ade7722db159814bb1a1b4bae47b21R150)

Reply:
Added both `init` and `initWithWindowScene`